sale_order_mass_action/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1 @@
+* Denis Roussel <denis.roussel@acsone.eu>
```

Comment:
could you add me ? ;-)
``* Sylvain LE GAL (https://twitter.com/legalsylvain)``

Reply:
Of course, forgot that part